### PR TITLE
Added the Baldeagle minecraft server.

### PR DESCRIPTION
**Pull request by SniverDaBest**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [ ] The file is in the `domains` folder and is in the JSON format.
- [ ] The file's name is all lowercased and alphanumeric. 
- [ ] You have completed your website. 
- [ ] The website is reachable.  
- [ ] The CNAME record doesn't contain `https://` or `/`.  
- [ ] There is sufficient information at the `owner` field.

### CODE REVIEW

File: domains/baldeagle.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "SniverDaBest",
+        "discord": "919385800364425256"
+    },
+    "record": {
+        "URL": "www.youtube.com/watch?v=dQw4w9WgXcQ"
```
Comment (CuteDog5695) [inline domains/baldeagle.json:7]: Very funny link (Won't merge until you change the link to something else)
```suggestion
        "URL": "https://www.youtube.com/watch?v=dQw4w9WgXcQ"
```

**Review by iostpa — CHANGES_REQUESTED** [review]

File: domains/baldeagle.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "SniverDaBest",
+        "discord": "919385800364425256"
+    },
+    "record": {
+        "URL": "sniver.serv00.net"
```
Comment (DEV-DIBSTER) [inline domains/baldeagle.json:7]: Make sure to have HTTPS or HTTP as a valid URL to redirect to.

File: domains/baldeagle.json [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "SniverDaBest",
+        "discord": "919385800364425256"
```
Comment (DEV-DIBSTER) [inline domains/baldeagle.json:4]: You need the email field or OWL field.

File: domains/mc.baldeagle.json [hunk 4]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "SniverDaBest",
+        "discord": "919385800364425256"
```
Comment (DEV-DIBSTER) [inline domains/mc.baldeagle.json:4]: Need email field or OWL field.

**Review by DEV-DIBSTER — CHANGES_REQUESTED** [review]

File: domains/baldeagle.json (hunk 3 above)
Comment (CuteDog5695) [reply domains/baldeagle.json:4]: Only email field is necessary though.

File: domains/mc.baldeagle.json (hunk 4 above)
Comment (CuteDog5695) [reply domains/mc.baldeagle.json:4]: Only email field is necessary though, OWL is optional.

**Pull request closed without merging**